rmgpy/qm/molecule.py

```diff
@@ -409,18 +409,41 @@ def parse(self):
         parser.logger.setLevel(
             logging.ERROR
         )  # cf. http://cclib.sourceforge.net/wiki/index.php/Using_cclib#Additional_information
-        parser.rotcons = (
-            []
-        )  # give it an attribute and it won't delete it, leaving it on the parser object
+parser.molmass = None # give it an attribute and it won't delete it, leaving it on the parser object
+parser.rotcons = (  # for cclib < 1.8.0
+    []
+)
+parser.rotconsts = (  # for cclib >= 1.8.0
+    []
+)
+cclib_data = parser.parse()  # fills in either parser.rotcons or parser.rotconsts but not both
+assert bool(parser.rotconsts) != bool(parser.rotcons)
+if parser.rotcons:  # for cclib < 1.8.0
+    cclib_data.rotcons = (
+        parser.rotcons
+    )
+else:  # for cclib >= 1.8.0
+    cclib_data.rotconsts = (
+        parser.rotconsts
+    )
         parser.molmass = None  # give it an attribute and it won't delete it, leaving it on the parser object
         cclib_data = parser.parse()
+
+        # for bacwards compatibility with cclib <1.6
+        if not hasattr(cclib_data, "rotconsts"):
+             # this hack required because rotconsts not part of a default cclib data object
+            parser.rotconsts = (
+                []
+            )  # give it an attribute and it won't delete it, leaving it on the parser object
+
+            cclib_data.rotconsts = (
+                parser.rotconsts
+            ) 
+
```

Comment:
I couldn't figure out how to do this as an inline suggestion earlier, but this section should be deleted. 
Note the `parser.molmass = None` line has been moved up to line 412, which is why I suggest deleting it here.
```suggestion
```